### PR TITLE
#19 infrastructure for service mesh

### PR DESCRIPTION
**Pull request by Emily-Jiang**
Signed-off-by: Emily Jiang <emijiang@uk.ibm.com>

### CODE REVIEW

File: spec/NOTICE [hunk 1]
```diff
@@ -0,0 +1,14 @@
+=========================================================================
+==  NOTICE file corresponding to section 4(d) of the Apache License,   ==
+==  Version 2.0, in this case for Microprofile Config                  ==
```
Comment (kenfinnigan) [inline spec/NOTICE]: Config?

File: spec/NOTICE [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+=========================================================================
+
+SPDXVersion: SPDX-2.1
+PackageName: Eclipse Microprofile
+PackageHomePage: http://www.eclipse.org/microprofile
+PackageLicenseDeclared: Apache-2.0
+
+PackageCopyrightText: <text>
+Mark Struberg struberg@apache.org
```
Comment (kenfinnigan) [inline spec/NOTICE]: Remove Mark?

File: spec/src/main/asciidoc/license-alv2.asciidoc [hunk 3]
```diff
@@ -0,0 +1,43 @@
+//
+// Copyright (c) 2016-2017 Eclipse Microprofile Contributors:
```
Comment (kenfinnigan) [inline spec/src/main/asciidoc/license-alv2.asciidoc]: 2018?

Remove Mark?

File: spec/src/main/asciidoc/license-alv2.asciidoc [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+Specification: {doctitle}
+
+Version: {revnumber}
+
+Status: {revremark}
+
+Release: {revdate}
+
+Copyright (c) 2016-2017 Contributors to the Eclipse Foundation
```
Comment (kenfinnigan) [inline spec/src/main/asciidoc/license-alv2.asciidoc]: 2018?

File: spec/src/main/asciidoc/microprofile-service-mesh-spec.asciidoc [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+= Microprofile Service Mesh
+:authors: Emily Jiang, Scott Clarks
```
Comment (kenfinnigan) [inline spec/src/main/asciidoc/microprofile-service-mesh-spec.asciidoc]: Who is Scott Clarks?

**Review by kenfinnigan — CHANGES_REQUESTED** [review]

File: spec/pom.xml [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+    </parent>
+
+    <groupId>org.eclipse.microprofile.service.mesh</groupId>
+    <artifactId>microprofile-service-mesh-spec</artifactId>
+    <packaging>pom</packaging>
+    <name>MicroProfile Service Mesh Specification</name>
+    <description>MicroProfile Service Mesh Specification :: Specification</description>
+
+    <properties>
+        <asciidoctor-maven.version>1.5.5</asciidoctor-maven.version>
```
Comment (pilhuhn) [inline spec/pom.xml]: Metrics uses 1.5.6

File: spec/NOTICE (hunk 1 above)
Comment (Emily-Jiang) [reply spec/NOTICE]: updated

File: spec/src/main/asciidoc/license-alv2.asciidoc (hunk 3 above)
Comment (Emily-Jiang) [reply spec/src/main/asciidoc/license-alv2.asciidoc]: updated

File: spec/src/main/asciidoc/license-alv2.asciidoc (hunk 4 above)
Comment (Emily-Jiang) [reply spec/src/main/asciidoc/license-alv2.asciidoc]: updated

File: spec/src/main/asciidoc/microprofile-service-mesh-spec.asciidoc (hunk 5 above)
Comment (Emily-Jiang) [reply spec/src/main/asciidoc/microprofile-service-mesh-spec.asciidoc]: :o corrected

File: spec/pom.xml (hunk 6 above)
Comment (Emily-Jiang) [reply spec/pom.xml]: updated to use 1.5.6

**Comment by Emily-Jiang** [comment]
Thanks @kenfinnigan @pilhuhn for your review comments! I think I have addressed them. Please check.

File: spec/NOTICE [hunk 7]
```diff
@@ -1,6 +1,6 @@
 =========================================================================
 ==  NOTICE file corresponding to section 4(d) of the Apache License,   ==
-==  Version 2.0, in this case for Microprofile Config                  ==
+==  Version 2.0, in this case for Microprofile Service Mes             ==
```
Comment (pilhuhn) [inline spec/NOTICE]: h missing

**Review by pilhuhn — CHANGES_REQUESTED** [review]

**Review by pilhuhn — APPROVED** [review]

**Comment by Emily-Jiang** [comment]
Thanks for your review @pilhuhn! @kenfinnigan I've addressed all of your comments. Please feel free to comment if you spot anything else.

**Pull request merged**